packages/query/src/price-indexer.ts

```diff
@@ -46,8 +46,9 @@ export const calculatePrice = (delta: Amount, unfilled: Amount, lambda: Amount):
  * This function processes only (1) price and ignores (2) price
  * We can get a BSOD with zero deltas(inputs), and we shouldn't save the price in that case
  */
-export const updatePrices = async (
+export const updatePricesFromSwaps = async (
```

Comment:
Wanted to make it clear that this helper is specifically for the swaps use case, not for e.g., delegation tokens.